x/oracle/client/cli/query.go

```diff
@@ -239,3 +240,32 @@ func GetCmdTxResponse() *cobra.Command {
 	flags.AddQueryFlagsToCmd(cmd)
 	return cmd
 }
+
+// GetCmdLeftBounty This function fetches the left bounty information for any given address.
+func GetCmdLeftBounty() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "left_bounty <address>",
```

Comment:
Should we use `left-bounty` for consistency ?

Reply:
sure